src/styles/bramble_overrides.css

```diff
@@ -76,3 +76,35 @@ li.jstree-leaf > a {
 .bramble-inspector-highlight {
     background: rgba(0, 135, 207, .3) !important;
 }
+
+/* Right-click File Menu */
+
+.context-menu .dropdown-menu  {
+  padding: 0;
+  border-radius: 0;
+  border: none;
+}
+
+.context-menu .dropdown-menu li a {
+  padding: 10px 12px;
+  line-height: 16px;
+  font-size: 16px;
+  cursor: pointer;
+}
+
+.context-menu .dropdown-menu li a:before {
```

Comment:
is this pseudo-element added somewhere else? That's why we need to override it here?